docs/skopeo-standalone-verify.1.md

```diff
@@ -30,6 +31,10 @@ Signature verified, digest sha256:20bf21ed457b390829cdbeec8795a7bea1626991fda603
 ## SEE ALSO
 skopeo(1), containers-signature(5)
 
+## NOTES
+
+As described in [Container signature format](https://github.com/containers/image/blob/main/docs/containers-signature.5.md), this command is intended for use with local signatures, such as OpenPGP. Further, this command does **not** interact with the artefacts generated by [Docker Content Trust (DCT)](https://docs.docker.com/engine/security/trust/).
```

Comment:
This link is not going to do me much good within a man page. You should expose the containers-signature.5 man page and reference it at the bottom.

Reply:
Ah, OK, good point - thanks - will do 👍

Reply:
```suggestion
As described in [Container signature format](https://github.com/containers/image/blob/main/docs/containers-signature.5.md), this command is intended for use with local signatures, such as OpenPGP. Further, this command does **not** interact with the artifacts generated by [Docker Content Trust (DCT)](https://docs.docker.com/engine/security/trust/).
```

Reply:
Thanks @TomSweeneyRedHat - based upon the kind feedback from other reviewers here, I've slimmed down the comments in `skopeo-copy` and `skopeo-standalone-sign` and `skopeo-standalone-verify` to reduce the verbiage, and instead reference out to [containers-signature](https://github.com/containers/image/blob/main/docs/containers-signature.5.md)

TBH I'd not fully appreciated that these docs were being used to generate the `man` pages for `skopeo` - my bad 😢